wire-ios-data-model/Source/Model/Conversation/ZMConversation+Creation.swift

```diff
@@ -109,7 +109,7 @@ extension ZMConversation {
                                                allowServices: Bool = true,
                                                readReceipts: Bool = false,
                                                participantsRole: Role? = nil) -> ZMConversation? {
-        return insertGroupConversation(moc: moc,
+        return insertConversation(moc: moc,
```

Comment:
Can we fix the indentation here? SwiftLint will complain. All properties need to align vertically.

Reply:
fixed